04/mobile.css

```diff
@@ -0,0 +1,88 @@
+body {
+    
+}
+.header__nav {
+    background-color: bisque;
+    height: 60px;
+    top: 0;
+    
+  
+}
+.nav {
+    position: absolute;
+    right: 0;
+    top: 9vh;
+    background-color: orange;
+    height: 300px;
```

Comment:
Bardzo bym uważał z ustawianiem pozycji menu na podstawie `vh` trzeba pamiętaj, że to wartość zależna od wielkości urządzenia. Jeśli ktoś na telefonie będzie oglądał w pionie stronę, a potem w poziomie to menu będzie w różnych miejscach. Jeśli t zamierzone to ok. Trzeba po prostu być tego świadomy :)